docs/help-center/your-account/account-settings.mdx

```diff
@@ -351,6 +351,39 @@ If you want to make sure noone can read your messages because you might forget t
 
 Here you will find all the devices connected to your account. You have the option to lock or remove devices here. Please note that by signing out of other devices, you will lose access to them. You will need to sign in again on those devices.
 
+## Screenshots
+
+On Android devices, you can enable or disable the ability to take screenshots. By default, it is disabled.  
+On iOS taking screenshots is enabled by default and will only give you a warning every time you take a screenshot
+therefore you will only find this setting on Android. 
+
+### Enable taking screenshot
+
+Please be careful with sensitive data. Preferably do not take screenshots of chats since that might leak sensitive data. 
+If you have to take a screenshot that might contain sensitive data, please delete it from your device when you do not
+need it anymore. Only share screenshots that contain personal data with people who are allowed to receive this data.  
+
+<Tabs>
+  <TabItem value="android" label="Android">
+  <ol>
+    <li>Tap on <b>Screenshots</b>.</li>
+    <li>If the switch is not active, tap on the switch.</li>
```

Comment:
You usually tap on a button and flip a switch. Not sure if this combination of wording is not common I think.